Build fails with Could not get unknown property 'fuzzOut' for project ':fuzzing

`./gradlew :tasks`

```
> Configure project :
Building Sigstore Java 0.4.0-SNAPSHOT

> Task :tasks FAILED

Build sigstore-java-root FAILURE reason:
    Execution failed for task ':tasks':
        org.gradle.api.internal.tasks.DefaultTaskContainer$TaskCreationException: Could not create task ':fuzzing:copyToFuzzOut'.
            at org.gradle.api.internal.tasks.DefaultTaskContainer.taskCreationException(DefaultTaskContainer.java:715)
            at org.gradle.api.internal.tasks.DefaultTaskContainer.access$600(DefaultTaskContainer.java:76)
            at org.gradle.api.internal.tasks.DefaultTaskContainer$TaskCreatingProvider.domainObjectCreationException(DefaultTaskContainer.java:707)
...
            Caused by: groovy.lang.MissingPropertyException: Could not get unknown property 'fuzzOut' for project ':fuzzing' of type org.gradle.api.Project.
                at org.gradle.internal.metaobject.AbstractDynamicObject.getMissingProperty(AbstractDynamicObject.java:85)
                at org.gradle.internal.metaobject.AbstractDynamicObject.getProperty(AbstractDynamicObject.java:62)
                at org.gradle.api.internal.project.DefaultProject.property(DefaultProject.java:1097)
                at Build_gradle$3.invoke(build.gradle.kts:16)
                at Build_gradle$3.invoke(build.gradle.kts:1)
                at org.gradle.api.internal.DefaultMutationGuard$2.execute(DefaultMutationGuard.java:44)
                at org.gradle.api.internal.DefaultMutationGuard$2.execute(DefaultMutationGuard.java:44)

```

--- a/fuzzing/build.gradle.kts
+++ b/fuzzing/build.gradle.kts
@@ -11,7 +11,10 @@ dependencies {
     implementation("com.code-intelligence:jazzer-api:0.16.0")
 }
 
+// copy to the fuzzing builder's output directory. This is an existing directory with
+// files in it, so don't use sync
 tasks.register<Copy>("copyToFuzzOut") {
     dependsOn(tasks.build)
-    into(project.property("fuzzOut")).from(sourceSets.main.get().runtimeClasspath)
+    into(project.findProperty("fuzzOut") ?: project.layout.buildDirectory.dir("fuzzOut"))
+    from(sourceSets.main.get().runtimeClasspath)
 }

